### PR TITLE
remove some un-used members

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: GitUI/Theming/Renderers/ButtonRenderer.cs [hunk 1]
```diff
@@ -280,21 +280,6 @@ public enum CheckBox
                 CBS_EXCLUDEDPRESSED = 19,
                 CBS_EXCLUDEDDISABLED = 20,
             }
-
```
Comment (gerhardol) [inline GitUI/Theming/Renderers/ButtonRenderer.cs:283]: Theme related, really remove?

File: BugReporter/Info/AssemblyInfo.cs [hunk 2]
```diff
@@ -1,12 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
-// <copyright file="AssemblyInfo.cs" company="NBug Project">
-//   Copyright (c) 2011 - 2013 Teoman Soygul. Licensed under MIT license.
-// </copyright>
-// --------------------------------------------------------------------------------------------------------------------
-
-namespace BugReporter.Info
-{
-    public class AssemblyInfo
```
Comment (mstv) [inline BugReporter/Info/AssemblyInfo.cs:9]: Are this and the following two `...Info` classes accessed using reflection in the BugReporter, @RussKie?

Comment (RussKie) [reply BugReporter/Info/AssemblyInfo.cs:9]: I don't think so... I did a quick search for these types, and I didn't see any uses. Let's nuke 'em :)

File: GitUI/Theming/Renderers/ButtonRenderer.cs (hunk 1 above)
Comment (RussKie) [reply GitUI/Theming/Renderers/ButtonRenderer.cs:283]: As an option, if we want to keep these, we can move all these unused theme-related types under `#if SUPPORT_THEME_HOOKS`.

Comment (SimonCropp) [reply GitUI/Theming/Renderers/ButtonRenderer.cs:283]: IMO u should remove them. then mark that commit with a tag. if u ever want to bring them back u can find them via the tag, and then revert the commit

Comment (RussKie) [reply GitUI/Theming/Renderers/ButtonRenderer.cs:283]: We've hidden all theme-related stuff behind `SUPPORT_THEME_HOOKS` pragma. Having all of those under the same pragma will make it easier to bulk-remove or bulk-reenable in the future.

Comment (gerhardol) [reply GitUI/Theming/Renderers/ButtonRenderer.cs:283]: I suggest this PR is changed to only include the first 5 removed files.

The theming related should be adding SUPPORT_THEME_HOOKS.
I still hope theming can be reenabled, then it is easier if the code is not removed.
However if removed, remove all in one dedicated commit, not many. There are probably other theming related methods that will be removed over time.

**Pull request closed without merging**